jinja2/tests.py

```diff
@@ -186,9 +205,21 @@ def test_in(value, seq):
     'iterable':         test_iterable,
     'callable':         test_callable,
     'sameas':           test_sameas,
-    'equalto':          test_equalto,
+    'in':               test_in,
     'escaped':          test_escaped,
-    'greaterthan':      test_greaterthan,
-    'lessthan':         test_lessthan,
-    'in':               test_in
+    '==':               test_compare_eq,
```

Comment:
I wouldn't use custom functions but `operator.eq` etc. - they do the same in the end but are faster.